src/s3_upload.py

```diff
@@ -48,23 +49,23 @@ def upload_todays_events_to_s3():
     """
     start_time = time.time()
 
-    print("Beginning upload of recent events to s3.")
+    logger.info("Beginning upload of recent events to s3.")
     fortyfive_min_ago = datetime.datetime.now() - datetime.timedelta(minutes=45)
```

Comment:
Do we need this anymore?

Reply:
yea, it doesnt really matter if we're intuiting the service date from the current time--the 30 minute interval wont ricochet us into a different date. i can remove!